app/views/non_hmis_assessments/pathways_version_four/_pathways_version_four_preamble.haml

```diff
@@ -0,0 +1,41 @@
+%p
+  Please read this 
+  %a{ href: 'https://bit.ly/pathways24intro', target: :_blank } bit.ly/pathways24intro
+  before completing the assessment
```

Comment:
Maybe I'm missing something, but I don't see this in the new form.  Can you confirm the text is correct in this section?

Reply:
This is showing in the header of the document, but i can remove it if it's not meant to be here.